pkg/metadata/sandbox.go

```diff
@@ -55,7 +67,9 @@ type SandboxMetadata struct {
 	// CreatedAt is the created timestamp.
 	CreatedAt int64
 	// NetNS is the network namespace used by the sandbox.
-	NetNS string
+	NetNS *SandboxNetNS
```

Comment:
Will send out a PR to refactor the metadata store soon. After that this will be much easier.